profiles/KaranChandekar.json

```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/93200960/174483769-0ec591f5-8438-4c97-8304-052a21f41254.png",
+    "issueId": 1191,
+    "name": "Karan Chandekar",
+    "username": "KaranChandekar"
+}
+  
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient